esx_service/cli/vmdkops_admin_test.py

```diff
@@ -211,7 +222,7 @@ def test_ls_no_args(self):
         volumes = vmdk_utils.get_volumes()
         header = vmdkops_admin.all_ls_headers()
         rows = vmdkops_admin.generate_ls_rows()
-        self.assertEqual(8, len(header))
+        self.assertEqual(10, len(header))
```

Comment:
same comment as above - a var with a name, instead of '10', would make it easier to read